src/app/searchResults/searchResults.component.js

```diff
@@ -12,29 +12,52 @@ let componentName = 'searchResults';
 class SearchResultsController {
 
   /* @ngInject */
-  constructor(designationsService) {
+  constructor($window, $location, designationsService) {
+    this.$window = $window;
+    this.$location = $location;
     this.designationsService = designationsService;
+    this.searchParams = {};
   }
 
   $onInit(){
-    let searchTerm = urlQueryTerm.get('q');
-    if(searchTerm){
-      this.designationsService.createSearch(searchTerm)
-        .subscribe((id) => {
-          this.searchId = id;
-          this.getPageResults(1);
+    var params = this.$location.search();
```

Comment:
We didn't have $locationProvider configured properly on production. Wasn't an issue with routers at all. You still need to use $window.location to reload or change the url, but we can use $location to read/write query params now.